__tests__/e2e/reference/slices/SlicifyHero/Hero/index.tsx

```diff
@@ -6,67 +6,65 @@ export type HeroProps = SliceComponentProps<Content.HeroSlice>
 
 export default function Hero({ slice }: HeroProps) {
 	return (
-		<section id={slice.primary.anchor || undefined}>
```

Comment:
just removed `anchor`, renamed the buttons property and remove the inner div that doesn't do anything